README.md

```diff
@@ -16,27 +16,24 @@ Before the build all dependencies must be downloaded and laid out as follows:
 
 ```
 <workspace>
-`-- llvm
-    |-- tools
-    |   `-- clang
-    `-- projects
-        |-- llvm-spirv
-        `-- opencl-clang
+|-- llvm
+|   `-- projects
```

Comment:
I think it is better to place llvm-spirv and opencl-clang on the top level as it is done in https://github.com/intel/llvm/

Reply:
Done